README.md

```diff
@@ -49,7 +49,7 @@ On Linux:
 ```console
 curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.14.0/kind-$(uname)-amd64"
 chmod +x ./kind
-mv ./kind /some-dir-in-your-PATH/kind
+sudo mv ./kind /usr/local/bin/kind
```

Comment:
All instances should be updated. There are at least lines 75 and 82 in this file and lines 74 and 81 in `site/content/docs/user/quick-start.md`

Reply:
Line 74 is for Mac & line 82 is for windows. I'm not a user of macOS/windows yet; hence I'm not sure of their default preferred locations. So, I made changes only to Linux installation.

Reply:
we can leave discussion on changing those for a different PR, IIRC macOS SIP makes it more problematic to suggest installing to system folders vs a user local folder added to path (like ~/go/bin)

Reply:
Sounds like a plan! There might be considerable discrepancies regarding macOS & windows default install locations.